mkosi/resources/mkosi-tools/mkosi.conf.d/azure-centos-fedora/mkosi.conf.d/efi.conf

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Architecture=uefi
+
+[Content]
+Packages=
+        grub2-tools
+        sbsigntools
```

Comment:
This will now be installed even when the EPEL repositories are not enabled, which will fail. sbsigntools can only be installed when the epel repositories are enabled. In other words the [Match] section from 20-epel-packages-9.conf can't just be dropped